packages/auth-bundler/tests/utils/client.ts

```diff
@@ -0,0 +1,30 @@
+import { faker } from '@faker-js/faker';
+import { IClient } from '@map-colonies/auth-core';
+
+export function getFakeClient(includeGeneratedFields: boolean): IClient {
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.lastName();
+  const client: IClient = {
+    name: faker.internet.userName(firstName, lastName),
```

Comment:
what about names with special characters ? we should also include & % _ - * $ and such.
We already have a client name with & in its name

Reply:
the file was not used in the tests so i removed it.